Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG CUDA_VERSION="11.8.0"
+ARG CUDNN_VERSION="8"
```

Comment:
have you tested without CUDNN? this is a purely curiousity question, i tried both CUDNN and regular devel and saw no difference for text-generation-webui

Reply:
I'm my testing, cuDNN was not required to build llama.cpp.

Reply:
cudnn isnt required .. correct - but this is premature optimisation for a temp solution